Move prover returns: exiting with boogie verification errors
error: post-condition does not hold

    ┌── tests/sources/functional/consts.move:29:9 ───
    │
 29 │         ensures !result.b;
    │         ^^^^^^^^^^^^^^^^^^
    │
    =     at tests/sources/functional/consts.move:24: init_incorrect
    =         result = <redacted>
    =     at tests/sources/functional/consts.move:25: init_incorrect
    =     at tests/sources/functional/consts.move:28
    =     at tests/sources/functional/consts.move:29
